.github/workflows/run-tests.yml

```
@@ -10,37 +10,29 @@ jobs:
     steps:
       - uses: actions/checkout@v4
 
-      - name: Set up Python
-        uses: actions/setup-python@v5
-        with:
-          python-version: "3.10"
-
-      - name: Install dependencies
-        run: |
-          python -m pip install -U pip
-          pip install -U pdm
-          pdm install
+      - name: Install uv
+        uses: astral-sh/setup-uv@v5
 
```

```suggestion
      - name: Install uv
        uses: astral-sh/setup-uv@v5
        with:  
          version: "0.6.0"

      - name: Set up Python
        uses: actions/setup-python@v5
        with:
          python-version: "3.12"

      - name: Install dependencies
        run: |
          uv sync --no-group docs
```
I think we need to install packages, at least for pytest, correct?